### PR TITLE
Removed "TODO: Not yet implemented as fixture" printouts

### PR DESCRIPTION
**Pull request by marcusnaslund**
In accordance with #620 
I will instead create issues for turning these tests into something using `Fixture`.
Anything to say @davidhesselbom ?

### CODE REVIEW

File: test/base/SynchronizedTest.ooc [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
-	refCounter: ReferenceCounter
-	init: func {
-		refCounter = ReferenceCounter new(this)
+version(debugTests) {
+	TestClass: class {
+		refCounter: ReferenceCounter
+		init: func {
+			refCounter = ReferenceCounter new(this)
+		}
+		dispose: func {
```
Comment (davidhesselbom) [inline test/base/SynchronizedTest.ooc]: `free`?

Comment (davidhesselbom) [inline test/base/SynchronizedTest.ooc]: Is this function even called from anywhere?

Comment (marcusnaslund) [inline test/base/SynchronizedTest.ooc]: This whole test is weird. I don't know why the diff is so crazy in this file. Do `?w=1` and you'll see it's not this PR.

File: test/base/SynchronizedTest.ooc [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
-func1: func {
-	for (i in 0 .. 10) {
-		obj1 := obj
-		Time sleepMilli(100)
+	
+	func2: func {
+		for (i in 0 .. 10) {
+			obj2 := obj
+			Time sleepMilli(76)
+		}
```
Comment (davidhesselbom) [inline test/base/SynchronizedTest.ooc]: And isn't it a little strange to define these functions after they were used? Consider moving these lines up to before the other things happen.

File: test/base/SynchronizedTest.ooc (hunk 1 above)
Comment (davidhesselbom) [inline test/base/SynchronizedTest.ooc]: I know, I just don't know if we should even keep the test.

Comment (marcusnaslund) [inline test/base/SynchronizedTest.ooc]: I will create an issue.

Comment (marcusnaslund) [inline test/base/SynchronizedTest.ooc]: #886

Comment (davidhesselbom) [inline test/base/SynchronizedTest.ooc]: Very well.

Comment (davidhesselbom) [inline test/base/SynchronizedTest.ooc]: So you should probably open issues for the other tests you just removed the printouts from, too.

Comment (marcusnaslund) [inline test/base/SynchronizedTest.ooc]: Right. Coming up.

Comment (marcusnaslund) [inline test/base/SynchronizedTest.ooc]: #887

**Pull request merged**